tutorial/source/tracking_1d_em.ipynb

```diff
@@ -183,21 +183,16 @@
    "outputs": [],
    "source": [
     "import heapq\n",
-    "def merge_objects(assignment, states_loc, threshold):\n",
-    "    exist_dist_probs = assignment.exists_dist.probs\n",
-    "    spurious_prob = assignment.assign_dist.probs[...,-1:]\n",
-    "    assign_dist_probs = assignment.assign_dist.probs[...,:-1].permute(2,0,1)\n",
+    "def prune_objects(states_loc, p_exist, p_exist_threshold = 1e-5):\n",
+    "        return states_loc[p_exist > p_exist_threshold]\n",
+    "\n",
+    "def merge_objects(states_loc, radius):\n",
     "    distance2 = (states_loc.unsqueeze(-2) -\n",
```

Comment:
If this dense `O(n^2)` op is too slow, you could instead compute sparse distances after initially populating the LSH. The sparse version will be slower for small problems, so I'd only refactor if it actually comes up as a speed issue.